File: src/com/basmilius/ps/bastools/identity/liveTemplates/IdentityLiveTemplatesProvider.java
package com.basmilius.ps.bastools.identity.liveTemplates;

import com.intellij.codeInsight.template.impl.DefaultLiveTemplatesProvider;
import org.jetbrains.annotations.Nullable;

public class IdentityLiveTemplatesProvider implements DefaultLiveTemplatesProvider
{

	@Override
	public String[] getDefaultLiveTemplateFiles ()
	{
		return new String[] { "liveTemplates/IdentityFramework" };
	}

	@Nullable
	@Override
	public String[] getHiddenLiveTemplateFiles ()
	{
		return new String[0];
	}

}

File: src/com/basmilius/bastools/framework/identity/liveTemplates/IdentityLiveTemplatesProvider.kt
/*
 * Copyright © 2018 Bas Milius <bas@mili.us> - All rights reserved.
 *
 * This file is part of Bas Tools, a plugin for the IntelliJ Platform.
 *
 * For the full copyright and license information, please view the
 * LICENSE file that was distributed with this source code.
 */

package com.basmilius.bastools.framework.identity.liveTemplates

import com.intellij.codeInsight.template.impl.DefaultLiveTemplatesProvider

/**
 * Class IdentityLiveTemplatesProvider
 *
 * @author Bas Milius <bas@mili.us>
 * @package com.basmilius.bastools.framework.identity.liveTemplates
 * @since 1.0.0
 */
class IdentityLiveTemplatesProvider: DefaultLiveTemplatesProvider
{

	/**
	 * {@inheritDoc}
	 *
	 * @author Bas Milius <bas@mili.us>
	 * @since 1.0.0
	 */
	override fun getDefaultLiveTemplateFiles() = arrayOf("liveTemplates/IdentityFramework")

	/**
	 * {@inheritDoc}
	 *
	 * @author Bas Milius <bas@mili.us>
	 * @since 1.0.0
	 */
	override fun getHiddenLiveTemplateFiles(): Array<String?> = arrayOfNulls(0)

}
